tests/func/test_config.py

```diff
@@ -95,7 +95,7 @@ def test_merging_two_levels(dvc):
         conf["remote"]["test"] = {"url": "ssh://example.com"}
 
     with dvc.config.edit("local") as conf:
-        conf["remote"]["test"] = {"password": "1"}
+        conf["remote"]["test"] = {"url": "ssh://example.com", "password": "1"}
```

Comment:
Actually this test was correct :) It checks if dvc will be able to merge config levels.

Reply:
Ok, I will make the config merging correct to pass this test.